### PR TITLE
Pagination

### PR DESCRIPTION
**Pull request by saidaipparla**

### CODE REVIEW

File: web/client/actions/maps.js [hunk 1]
```diff
@@ -178,7 +178,7 @@ function permissionsLoaded(permissions, mapId) {
     };
 }
 
-function loadMaps(geoStoreUrl, searchText="*", params={start: 0, limit: 20}) {
+function loadMaps(geoStoreUrl, searchText="*", params={start: 0, limit: 5}) {
```
Comment (mbarto) [inline web/client/actions/maps.js]: Why are we changing the limit to 5?

File: web/client/plugins/Maps.jsx [hunk 2]
```diff
@@ -60,9 +60,9 @@ const PaginationToolbar = connect((state) => {
     }
     let {start, limit, results, loading, totalCount, searchText} = state.maps;
     let page = 0;
-    let total = totalCount || 0;
+    //    let total = totalCount || 0;
     if (results && totalCount) { // must be !==0 and exist to do the division
-        page = Math.ceil(start / total);
+        page = Math.ceil(start / limit);
```
Comment (mbarto) [inline web/client/plugins/Maps.jsx]: We should probably compare limit and totalCount and take the lowest?

File: web/client/actions/maps.js (hunk 1 above)
Comment (saidaipparla) [inline web/client/actions/maps.js]: i changed for  easy  debugging  and i changed back in second commit.

File: web/client/plugins/Maps.jsx [hunk 3]
```diff
@@ -60,11 +60,16 @@ const PaginationToolbar = connect((state) => {
     }
     let {start, limit, results, loading, totalCount, searchText} = state.maps;
     let page = 0;
-    //    let total = totalCount || 0;
-    if (results && totalCount) { // must be !==0 and exist to do the division
-        page = Math.ceil(start / limit);
+    let total = totalCount || 0;
```
Comment (mbarto) [inline web/client/plugins/Maps.jsx]: const total = Math.min(totalCount || 0, limit || 0);
const page = (results && total && Math.ceil(start / total)) || 0;

Isn't this simpler?

Let's use const as much as possible. Let is usually not necessary.

**Pull request merged**